dagger/modeler/builder_helper.py

```diff
@@ -141,7 +141,7 @@ def generic_command_task_builder(
             value_serializer=value_serializer,
         )
         command_task_builder = KafkaCommandTaskTemplateBuilder(self.app)
-        command_task_builder.set_topic(command_topic)
+        command_task_builder.set_topic(command_topic)  # type: ignore
```

Comment:
Why do we need to ignore the type here now?

Reply:
TopicT to Topic conversion